docs/scenes/get-started/set-up-your-environment/instructions/iosPhysicalDevelopmentBuildLocal.mdx

```diff
@@ -50,4 +50,6 @@ Run the following command in your project's root directory and select your plugg
 
 > This command runs a development server after building your app. You can skip running `npx expo start` on the next page.
 
+> Update `bundleIdentifier` of `app.json` in the root directory to a unique value, or Xcode won’t properly generate the provisioning profile for signing, causing the launch failure.
```

Comment:
```suggestion
> Add the `ios.bundleIdentifier` in the **app.json** file in the root directory to a unique value so that Xcode generates the provisioning profile for app signing.
```

- Suggestion to improve the callout and make it more positive.
- Can you move this callout at the top of this section so that the value is set before the command runs.